### ISSUE TITLE
Error extracting additional include types

### ISSUE THREAD

**Issue by jainankit** [os72/protoc-jar-maven-plugin#74]
We are using a multi-module configuration for maven where the base module looks like:

```
<modules>
    <module>server</module>
    <module>offline</module>
    <module>bgservice</module>
  </modules>
```

and the bgservice is the one that includes the `protoc-jar-maven-plugin`:

```
<plugin>
        <groupId>com.github.os72</groupId>
        <artifactId>protoc-jar-maven-plugin</artifactId>
        <version>3.7.0.2</version>
        <executions>
          <execution>
            <phase>generate-sources</phase>
            <goals>
              <goal>run</goal>
            </goals>
            <configuration>
              <protocArtifact>com.google.protobuf:protoc:3.0.0</protocArtifact>
              <includeMavenTypes>direct</includeMavenTypes>
              <includeDirectories>
                <include>src/main/proto</include>
              </includeDirectories>
              <inputDirectories>
                <include>src/main/proto</include>
              </inputDirectories>
              <outputTargets>
                <outputTarget>
                  <type>java</type>
                </outputTarget>
                <outputTarget>
                  <type>grpc-java</type>
                  <pluginArtifact>io.grpc:protoc-gen-grpc-java:1.0.1</pluginArtifact>
                </outputTarget>
              </outputTargets>
            </configuration>
          </execution>
        </executions>
      </plugin>
```

Interestingly, when running from base directory, the `mvn clean install` command works but `mvn compile` fails with error:

```
[ERROR] Failed to execute goal com.github.os72:protoc-jar-maven-plugin:3.7.0.2:run (default) on project testproject-bgservice: Error extracting additional include types: /Users/ankit/testproject/server/target/classes (Is a directory)
```

Here's the debug error log:
```
Caused by: java.io.FileNotFoundException: /Users/ankit/testproject/server/target/classes (Is a directory)
    at java.io.FileInputStream.open0 (Native Method)
    at java.io.FileInputStream.open (FileInputStream.java:195)
    at java.io.FileInputStream.<init> (FileInputStream.java:138)
    at com.github.os72.protocjar.maven.ProtocJarMojo.extractProtosFromDependencies (ProtocJarMojo.java:515)
    at com.github.os72.protocjar.maven.ProtocJarMojo.performProtoCompilation (ProtocJarMojo.java:381)
    at com.github.os72.protocjar.maven.ProtocJarMojo.execute (ProtocJarMojo.java:353)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
[ERROR]
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :testproject-bgservice
```

For now, I've removed the `<includeMavenTypes>direct</includeMavenTypes>` and added the includeStdTypes that avoids this issue:
```
<protocVersion>3.0.0</protocVersion>
<includeStdTypes>true</includeStdTypes>
```
But still unsure of the root cause.

**Comment by os72** [comment]
Could you pull and build locally and try 3.7.1-SNAPSHOT?
Somehow  `server/target/classes` becomes a dependency artifact for your module (?)

**Comment by os72** [comment]
Released in version 3.7.1
